demos/general/demo_OnACID_mesoscope.py

```diff
@@ -101,10 +99,18 @@ def main():
     opts = cnmf.params.CNMFParams(params_dict=params_dict)
 
 # %% fit online
-
+    init_file_save = fnames[0][:-4]+'_init.hdf5'
     cnm = cnmf.online_cnmf.OnACID(params=opts)
-    cnm.fit_online()
-
+    if os.path.exists(init_file_save):
```

Comment:
including this feature as in the main demo add some complexity. Perhaps there is a better way.